static/index.html

```diff
@@ -5,6 +5,8 @@
     <meta charset="utf-8">
     <title>Falcon SQL Client</title>
 
+    <link rel="stylesheet" type="text/css" href="static/css/falcon.css"></link>
```

Comment:
Probably too late to change; however, is there a reason we have separate css files and not using something webpack to bundle this.  Again not important just an observation

Reply:
@n-riesco Looks good as well 💃

Reply:
@shannonlal 

> Probably too late to change; however, is there a reason we have separate css files and not using something webpack to bundle this. Again not important just an observation

This is a temporary solution. I'd like to come back to packing CSS with webpack and possibly using http://lesscss.org/ to define a some sort of theme to be used by all components.

Reply:
>  I'd like to come back to packing CSS with webpack and possibly using http://lesscss.org/ to define a some sort of theme to be used by all components.

I've heard good things about [Tachyons](http://tachyons.io/), which seems to be what [SQLPad is using](https://github.com/rickbergfalk/sqlpad/pull/274). Should be easy enough to harmonize something like this with plotly's [styleguide](https://brand.plot.ly/).

Reply:
Jack, as usual, your collections of links are gold! :)